index.html

```diff
@@ -68,39 +68,45 @@
     <script src="https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js"> </script>
     <script>
       // $(document).foundation();
+      // Default branch is master
+      // TODO: change to branch prod when all are available
+      var branch_name = 'master'
+      var urlParms = new URLSearchParams(location.search)
+      var has_branch = urlParms.has('branch')
+      if (has_branch) branch_name = urlParms.get('branch')
```

Comment:
I think this could be simplified in a ternary expression like:

var branch_name = urlParams.has('branch') ? urlParams.get('branch') : 'master'